todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/statistics/ui/StatisticsViews.kt

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2016, The Android Open Source Project
+ * Copyright (c) 2017-2018 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.example.android.architecture.blueprints.todoapp.statistics.ui
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import com.example.android.architecture.blueprints.todoapp.R
+import com.example.android.architecture.blueprints.todoapp.statistics.StatisticsEvent
+import com.example.android.architecture.blueprints.todoapp.statistics.StatisticsState
+import com.spotify.mobius.Connectable
+import com.spotify.mobius.Connection
+import com.spotify.mobius.functions.Consumer
+
+class StatisticsViews(inflater: LayoutInflater, parent: ViewGroup) : Connectable<StatisticsState, StatisticsEvent> {
+
+    val rootView: View = inflater.inflate(R.layout.statistics_frag, parent, false)
```

Comment:
Should this be private too? Also, is it possible to initialize in the init block for consistency? I'm not sure yet when to initialize in an init block vs inline like that, do you have any ideas?

Reply:
But then wouldn't inlining the initialization of `statisticsTV` follow the same logic?